synapse/handlers/room.py

```diff
@@ -1068,6 +1019,13 @@ async def _send_events_for_new_room(
             A tuple containing the stream ID, event ID and depth of the last
             event sent to the room.
         """
+        visibility = room_config.get("visibility", "private")
```

Comment:
`visibility` seems unused here?

Reply:
It's used right below, i.e. `if visibility == "private"`

Reply:
Doh, so it is. 🤦